doc/api/cluster.md

```diff
@@ -198,8 +198,11 @@ Within a worker, `process.on('message')` may also be used.
 
 See [`process` event: `'message'`][].
 
-As an example, here is a cluster that keeps count of the number of requests
-in the master process using the message system:
+Here is an example using the message system. It keeps a count in the master
+process of the number of HTTP requests received by the workers:
+
+Note that when this example running, you need to open browser and visit `http://localhost:8000`
+or open another terminal window and run `curl http://localhost:8000` to see `numReqs` increasing.
```

Comment:
I don't think the new sentence is necessary with the changes to the existing sentence.